primer_designer/templates/primer_designer/index.html

```diff
@@ -1,107 +1,74 @@
-
-      <br></br>
-
-      <!-- Input form -->
-      <form role="form"  id="panel_form" method="post" onsubmit="return spinner_function()">
-        {% csrf_token %}
-        <div class="input-group">
-          {{regions_form.regions}}
-        </div>
-        <br>
-        <div class="input-group">
-          <span class="input-group-btn">
-            <button class="btn alert-success" type="submit">Design primers</button>
-          </span>
-        </div>
-      </form>
-  </div>
-
-  <!-- loading spinner -->
-  <div id="spinner" class="spinner" style="display:none">
-    <span style="vertical-align:middle">Designing primers...</span>
-    <img id='spinner' src='/static/images/dna_loader.gif' alt="Please wait .."  style="width:300px;height:165px; vertical-align:middle">
+{% block body %}
+{% load crispy_forms_tags %}
+<div class="container">
+  <div class="card p-5 my-5">
+    <div class="card-body">
+      <h2> Primer Designer</h2>
+      <div>
+          Enter regions to design primers for (one region per line)
+    
+          <br></br>
+    
+          Example input formats:<br>
+    
+          <ul>
+            <li class="my-2">Single Position: <pre class="inline px-2">x:123213 grch37</pre></li>
+            <li class="my-2">Range Position: <pre class="inline px-2">x:123213-547363 grch37</pre></li>
+            <li class="my-2">Fusion Design: <pre class="inline px-2">1:125432:b:1 2:2635462:a:1 grch37</pre></li>
+          </ul>
+    
+          For fusion designs, each should be in the format of <pre class="inline">chr:pos:side:strand</pre> for both regions,<br>
+          followed by the <pre class="inline">grch37</pre> or <pre class="inline">grch38</pre>.<br></br>
+          <pre class="inline">side</pre> should be either <pre class="inline">a/b</pre> and <pre class="inline">strand</pre>
+          should be either <pre class="inline">1/-1</pre> and denotes where to design primers relative to a breakpoint.
+          
+          <br></br>
+          <p>Example: 1:1625342 grch37</p>
+    
+          <!-- Input form -->
+          <form id="primer_form" clas="my-5" method="POST">
+            {% csrf_token %}
+            {{ regions_form|crispy }}
+            <div class="row justify-content-between px-3">
+              <button id="primer_submit" type="submit" class="btn btn-primary btn-sm">Submit</button>
+              <a id="primerHome" class="btn btn-info btn-sm" href="/" role="button">Home</a>
+            </div>
+          </form>
+      </div>
+
+      <!-- loading spinner -->
+      <div id="spinner" style="display:none;">
+        <span style="vertical-align:middle">Designing primers...</span>
+        <img id='spinner' src='/static/images/dna-loader.gif' alt="Please wait .."  style="width:300px;height:165px; vertical-align:middle">
+      </div>
+    </div>
   </div>
 </div>
-
 {% endblock %}
 
 
 {% block script %}
-
-$(function() {
```

Comment:
not sure why this is still here, this is from the current old live mess we have now and should have gone...

Reply:
it's deleted (highlighted red)